mac

```diff
@@ -181,6 +181,22 @@ install_asdf_language() {
   fi
 }
 
+install_asdf_language_version() {
+  local language="$1"
+  local version="$2"
+
+  if ! asdf list "$language" | grep -Fq "$version"; then
+    asdf install "$language" "$version"
+    asdf global "$language" "$version"
+  fi
+}
+
+start_postgres_server() {
+  if ! pg_ctl status | grep -Fq "server is running"; then
+    pg_ctl start -l "$HOME/postgres.log"
```

Comment:
What is the advantage of changing where logs go? My worry is now if you start postgres any other way, you have to either remember to specify where you want the lgos, or have logs in two different places.

Reply:
By default, `pg_ctl` will log all output to your terminal window. The only way to get it in a file is to specify where it goes, either by redirecting standard output or giving the `-l` option.